internal/services/mssql/helper/sql_retention_policies.go

```diff
@@ -70,6 +70,11 @@ func ShortTermRetentionPolicySchema() *pluginsdk.Schema {
 					Required:     true,
 					ValidateFunc: validation.IntBetween(1, 35),
 				},
+				"backup_interval_in_hours": {
```

Comment:
It looks like Azure defaults this value to 12 so we'll want to do the same here

Reply:
Thanks, I have fixed that.